examples/cockroachdb/cockroachdb-statefulset.yaml

```diff
@@ -66,6 +78,21 @@ spec:
       labels:
         app: cockroachdb
       annotations:
+        scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
What happens if you scale the statefulset to run more pods than nodes? @smarterclayton @bprashanth do we have an issue somewhere about adding Conditions in StatefulSets and surface any failures?